core/modules/user/user.admin.inc

```diff
@@ -399,6 +399,32 @@ function user_login_settings($form, &$form_state) {
     ),
   );
 
+  $form['user_login_appearance_setting'] = array(
+    '#type' => 'fieldset',
+    '#title' => '',
+  );
+  $user_login_page_tab_options = array(
+    'tabs' => t('Use tabs'),
+    'links' => t('Use links'),
+    'simplified' => t('Show simplified page'),
+  );
+  $user_login_page_default_settings = $core_config->get('user_login_appearance_setting');
+  $form['user_login_appearance_setting']['user_login_appearance_setting'] = array(
```

Comment:
I belief, this form (user_login_settings) needs an additional submit handler to flush related caches.

Reply:
Done.